app/src/main/java/com/sdp/swiftwallet/CryptoGraphActivity.java

```diff
@@ -36,8 +37,9 @@
 import java.util.ArrayList;
 
 public class CryptoGraphActivity extends AppCompatActivity {
-    private final int NB_CANDLES_TO_SHOW = 10;
+    private final int NB_CANDLES_TO_SHOW = 100;
```

Comment:
Check that it's not too much to display, Cirrus integration for UI testing behaves strangely with heavy layouts